File: plugins/time-keeper/src/main/java/jp/szlab/mcplugins/timekeeper/TimeKeeperPlugin.java
package jp.szlab.mcplugins.timekeeper;

import org.bukkit.plugin.java.JavaPlugin;

public class TimeKeeperPlugin extends JavaPlugin {
    @Override
    public void onEnable() {
        getLogger().info("TimeKeeper has been enabled.");
    }

    @Override
    public void onDisable() {
        getLogger().info("TimeKeeper has been disabled.");
    }
}

File: plugins/time-keeper/src/main/kotlin/jp/szlab/mcplugins/timekeeper/TimeKeeperPlugin.kt
package jp.szlab.mcplugins.timekeeper

import org.bukkit.plugin.java.JavaPlugin

class TimeKeeperPlugin : JavaPlugin() {
    override fun onEnable() {
        logger.info("TimeKeeper has been enabled.")
    }

    override fun onDisable() {
        logger.info("TimeKeeper has been disabled.")
    }
}
